stock_picking_reorder_lines/views/view.xml

```diff
@@ -3,7 +3,6 @@
     <data>
 
         <record id="view_move_picking_tree" model="ir.ui.view">
-            <field name="name">stock.move.tree</field>
```

Comment:
I feel convenient to name it something like stock.move tree.what.it.change

But ok thanks I learned something:
https://github.com/OCA/OCB/blob/8.0/openerp/addons/base/ir/ir_ui_view.py#L257-L258

Then I agree with you you can keep it empty